src/sssom/util.py

```diff
@@ -71,9 +72,6 @@
 
 logging = _logging.getLogger(__name__)
 
-#: The key that's used in the YAML section of an SSSOM file
-PREFIX_MAP_KEY = "curie_map"
```

Comment:
This line here is my only "concern". Could this be changed to PREFIX_MAP_KEY = CURIE_MAP for backwards compatibility, or not worth it? I leave the judgement to you.

Reply:
The issue is the proliferation of constants, so I would rather not